scripts/bert/export/hybrid_bert.py

```diff
@@ -530,7 +531,7 @@ def _apply_pooling(self, sequence):
 
         This is used for pre-training or fine-tuning a BERT model.
         """
-        outputs = sequence[:, 0, :]
+        outputs = sequence.slice(begin=(None, 0, None), end=(None, 1, None)).squeeze(axis=1)
         return self.pooler(outputs)
 
     def _decode(self, sequence, masked_positions):
```

Comment:
Because we won`t use the decoder at bert, so can we delete the decoder part or set 
> use_decoder : bool, default True
        Whether to include the decoder for masked language model prediction.

 default false? i think it will make our bert model clearer.

Reply:
Good observation. I try to be consistent with the get_bert_model in the gluonnlp API: https://github.com/dmlc/gluon-nlp/blob/master/src/gluonnlp/model/bert.py#L630 

It might be true that most people use BERT for finetuning and don't need the decoder. But it's not likely that we'll break existing API before gluonnlp 1.0.